### PR TITLE
Updated operators to print with hat

### PR DESCRIPTION
**Pull request by rbaliarizona**
Operators now print with hat on top

### CODE REVIEW

**Comment by rbaliarizona** [comment]
@Krastanov  my main goal here was to familiarize myself with the pull-request process. I personally like my operators to display with hats, but I know this isn't a law, so feel free to discard this change. Any feedback on if I went through the pull request process would be welcome though :)

**Comment by Krastanov** [comment]
This is great, Rohan, thanks! I will run a review on it later today

File: src/QSymbolicsBase/literal_objects.jl [hunk 1]
```diff
@@ -138,7 +138,7 @@ basis(x::SymQ) = x.basis
 
 Base.show(io::IO, x::SKet) = print(io, "|$(symbollabel(x))⟩")
 Base.show(io::IO, x::SBra) = print(io, "⟨$(symbollabel(x))|")
-Base.show(io::IO, x::Union{SOperator,SHermitianOperator,SUnitaryOperator,SHermitianUnitaryOperator,SSuperOperator}) = print(io, "$(symbollabel(x))")
+Base.show(io::IO, x::Union{SOperator,SHermitianOperator,SUnitaryOperator,SHermitianUnitaryOperator,SSuperOperator}) = print(io, "$(symbollabel(x))̂")
```
Comment (Krastanov) [inline src/QSymbolicsBase/literal_objects.jl:141]: Two points:

- we probably should insert the hat on the first letter (not all operators are a single symbol)

- let's remove `SSuperOperator` from this list for the moment, as it has a bit more complicated printing

**Review by Krastanov — CHANGES_REQUESTED** [review]
I love that the PR is small -- way easier to review and merge.

I left one inline comment on some code details.

There are a lot of failing tests. An easy way to run the tests locally and check things is to do `] test QuantumSymbolics` -- that way you can find exactly which test is failing and focus on it.

Most of the tests here seem to be "doctests" that just lack a hat. There is an automated way to fix them (the `fix` argument to `doctest` function).

Happy to discuss testing more at the hackathon.

**Comment by rbaliarizona** [comment]
@Krastanov thanks for the feedback!

I have updated the doctests. Hopefully they pass now. 

I actually added logic to make the hat print on the middle character instead. I know this wasn't your suggestion and it may be more logic that we are looking for here. Lmk what you think!

**Comment by Krastanov** [comment]
apologies, it seems I messed up the merge with master. Let me try to fix that

**Comment by Krastanov** [comment]
Darn, there are a few new places in the documentation where the hats need to be added. For now I will mark this as a draft as I will not have the time to make these changes. Apologies for the delay. Happy to pick it back up if you are interested in finishing it, but no pressure on that.

**Comment by rbaliarizona** [comment]
Hi @Krastanov , thanks for looking at it. Unfortunately I think I will need to drop this, atleast for now.

**Comment by Krastanov** [comment]
no worries!

**Pull request closed without merging**